components/search/src/filter.rs

```diff
@@ -183,8 +183,26 @@ mod tests {
                         }]),
                         search_term_param_name: Some("baz".to_string()),
                     },
-                    suggestions: None,
-                    trending: None,
+                    suggestions: Some(JSONEngineUrl {
+                        base: "https://example.com/suggestions".to_string(),
+                        method: Some(crate::JSONEngineMethod::Get),
+                        params: Some(vec![SearchUrlParam {
+                            name: "lang".to_string(),
+                            value: None,
+                            experiment_config: Some("lang-param".to_string()),
```

Comment:
How about changing the name and experiment_config? 
`lang` to `suggest-name`and `now` to `suggest-experiment-value` might be more clear?